cli/wallet/validator.go

```diff
@@ -158,3 +179,41 @@ func handleVote(ctx *cli.Context) error {
 		return contract.VoteUnsigned(addr, pub)
 	})
 }
+
+func handleGasAction(ctx *cli.Context, mkTx func(*neo.Contract, *nep17.Token, util.Uint160, *wallet.Account) (*transaction.Transaction, error)) error {
```

Comment:
There’s a lot of code duplication between this function and handleNeoAction. Can we squash these two functions to reuse existing code as much as possible?